### PR TITLE
zkvm: add `neg` instruction

### PR DESCRIPTION
**Pull request by decentralisedkev**
Added the Neg Instruction

### CODE REVIEW

**Comment by tessr** [comment]
Hi @decentralisedkev ! Thanks so much for all of your contributions to ZkVM. It’s a lot of fun to have an outside contributor :) 

My name is Tess, and I’m the manager for the team that works on ZkVM at Interstellar. I mostly wanted to say hello, but I also have an ask for you. So far you’re our only external (non-Interstellar) contributor, so it’s possible to have a bit of a collision if you’re working on something and someone else is working on the same thing! 

So, one idea: Would you mind opening a PR as soon as you start working on something, just to let the rest of us know what you’re up to? (It can even be a “draft PR,” a new feature that github just announced yesterday: https://github.blog/2019-02-14-introducing-draft-pull-requests/) 

What do you think?

**Comment by decentralisedkev** [comment]
@tessr Hey Tessa! My apologise, that sounds like a great idea. Should I close this PR?

**Comment by tessr** [comment]
No, you can leave it open! But if you could open a PR whenever you start on something new, that would be a great way to let us know what you're up to.

Thank you again for your contributions to ZkVM.

**Comment by tessr** [comment]
Oh and @oleganza or @vickiniu -- do one of you want to give this PR a quick review?

File: zkvm/src/vm.rs [hunk 1]
```diff
@@ -290,6 +290,15 @@ where
         Ok(())
     }
 
+    fn neg(&mut self) -> Result<(), VMError> {
+        let mut expr = self.pop_item()?.to_expression()?;
+        for (_, n) in expr.terms.iter_mut() {
+            -*n;
```
Comment (oleganza) [inline zkvm/src/vm.rs]: lol

**Pull request merged**